index.js

```diff
@@ -31,10 +35,11 @@ module.exports = function packager (opts, cb) {
     default: return cb(new Error('Unsupported platform. Must be either darwin, linux, or win32'))
   }
 
-  // Ignore this and related modules by default
-  var defaultIgnores = ['/node_modules/electron-prebuilt($|/)', '/node_modules/electron-packager($|/)', '/\.git($|/)']
-  if (opts.ignore && !Array.isArray(opts.ignore)) opts.ignore = [opts.ignore]
-  opts.ignore = (opts.ignore) ? opts.ignore.concat(defaultIgnores) : defaultIgnores
+  // Only include files specified
+  var defaultFiles = ['package.json']
+  opts.files = (defaultFiles + config.files).map(function (item) {
+    path.join(opts.dir, item)
```

Comment:
Also in general (sorry for the deluge of comments) I am open to the 'positive' approach taken here, but I'm not sure we should remove the old way of doing it (where you filter stuff out). I think before making any breaking changes it would be a good idea to open a discussion issue to get feedback from the other maintainers/users on what we think a good set of APIs would be.